src/tools/rbd_mirror/Mirror.cc

```diff
@@ -536,6 +536,10 @@ void Mirror::handle_signal(int signum)
 
 int Mirror::init()
 {
+  std::ostringstream oss;
+  std::copy(m_args.begin(), m_args.end(), std::ostream_iterator<string>(oss, " "));
+  dout(10) << "cmd-line arguments for rbd-mirror: " << oss.str() << dendl;
```

Comment:
This feels out of place here, since `m_args` isn't used in this method at all.  Perhaps put this in `Mirror::Mirror()` constructor?

Also, no need for the `std::copy()` business:
```
dout(10) << "args=" << args << dendl;
```

Reply:
@idryomov 

Without the `std:copy()`

```
2023-06-27T17:35:32.098+0530 7f37ab491840 10 rbd::mirror::Mirror: 0x55b41d57f600 Mirror: args=[--cluster,site-a,--rbd-mirror-remote-osd-op-timeout=5,--rbd-mirror-pool-replayers-refresh-interval=30,--rbd-mirror-image-state-check-interval=5,--debug-rbd=30,--debug-rbd_mirror=30] 
```

With `std:copy`, its well formatted

```
2023-06-27T17:35:32.098+0530 7f37ab491840 10 rbd::mirror::Mirror: 0x55b41d57f600 Mirror: args1=--cluster site-a --rbd-mirror-remote-osd-op-timeout=5 --rbd-mirror-pool-replayers-refresh-interval=30 --rbd-mirror-image-state-check-interval=5 --debug-rbd=30 --debug-rbd_mirror=30  
```

I will wait for your inputs though.

Reply:
We are dumping an argument which happens to be a vector, not a string.  It would be nice to have it formatted like a vector would be everywhere else in Ceph.

Reply:
Sure, keeping it aligned with other usages with in Ceph.